frontend/.eslintrc.js

```diff
@@ -28,6 +28,10 @@ module.exports = {
         "padded-blocks": "off",
         "quote-props": "off",
         "no-magic-numbers": "off",
+        "max-lines-per-function": "off",
+        "default-case": "off",
+        "no-console": "off",
+        "func-style": "off",
```

Comment:
Am I right in assuming that this triggered because of the `constructor`?

Reply:
Yes

Reply:
That's BS, but okay.